scripts/runHorusec.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+# Check for install/updates at https://github.com/ZupIT/horusec
+
+benchmark_version=$(scripts/getBenchmarkVersion.sh)
+horusec_version=$(horusec version 2>&1 | grep version | awk '{print $NF}')
+result_file="results/Benchmark_$benchmark_version-horusec-$horusec_version.json"
+
+horusec start -t 3600 -p="." -o="json" -O="$result_file"
```

Comment:
It looks like this is using the horusec-cli. Does that mean it is or is NOT using horusec with Docker? I ask because they say without Docker loses much of the analysis power.

Also, can you enhance this script to verify that horusec is even installed, and if it is not installed, either scripts up the installation and asks you if you want to install it, or at least provides basic install instructions, and then exits.

Reply:
Switched to docker.